lib/terraforming/resource/auto_scaling_group.rb

```diff
@@ -67,7 +67,17 @@ def tfstate
       private
 
       def auto_scaling_groups
-        @client.describe_auto_scaling_groups.map(&:auto_scaling_groups).flatten
+        token = ""
```

Comment:
I think we don't have to rewrite this code...

As described at https://aws.amazon.com/jp/blogs/developer/response-paging/ , pagination can be written more simply using loop.

```ruby
@client.describe_auto_scaling_groups.map(&:auto_scaling_groups).flatten
```

can be rewritten as:

```
asgs = []
@client.describe_auto_scaling_groups.each do |response|
  asgs << response.auto_scaling_groups
end.flatten
```